src/lib/components/Sort/Sort.js

```diff
@@ -83,6 +86,7 @@ Sort.propTypes = {
   label: PropTypes.func,
   overridableId: PropTypes.string,
   sortOrderDisabled: PropTypes.bool,
+  ariaLabel: PropTypes.string
```

Comment:
can you please add the `default` value just below, given that it is not a required `prop`?

Reply:
Empty string is not great for A11y.  I put in the default value of "Sort" even though it's not translated.  Will also make a PR for invenio-search-ui that includes translation option.